src/System.Text.RegularExpressions/ref/System.Text.RegularExpressions.cs

```diff
@@ -62,9 +62,22 @@ public partial class MatchCollection : System.Collections.ICollection, System.Co
         void System.Collections.ICollection.CopyTo(System.Array array, int arrayIndex) { }
     }
     public delegate string MatchEvaluator(System.Text.RegularExpressions.Match match);
+    abstract public partial class RegexRunnerFactory
```

Comment:
Increased the contract version to 4.1.0.0, but expect apicompat to throw error. Locally on build I didnt get any apicompat errors. @ericstj How do i verify apicompat? Should i be updating the buildtools ```baseline.packages.targets``` first to change ```System.Text.RegularExpressions``` from 4.0.12 to 4.1.0? I made that change locally on the targets in my Tools folder, but the apicompat didnt throw error even then..